src/components/ContactDetails.vue

```diff
@@ -392,6 +392,7 @@ export default {
 			this.fixed = false
 			this.loadingUpdate = true
 			await this.$store.dispatch('updateContact', this.localContact)
+			this.updateLocalContact()
```

Comment:
Doing this will erase any changes made during the firing of `updateContact` and the `updateLocalContact`. On small connexion this could cause some data loss. Like you're writing a note, and the update gets fired, but you keep writing. When the `updateLocalContact` is reached, everything you kept writing will go back to the original state of the contact (not local) :thinking: 

I need to check if my theory is good :)